webpack/strategies/docs.js

```diff
@@ -23,10 +23,10 @@ export default (config, options) => {
         noParse: /babel-core\/browser/,
         loaders: config.module.loaders
           .map(value => {
-            if (/\.js\/$/.test(value.test.toString())) {
+            if (/js/.test(value.test.toString())) {
```

Comment:
It seems I miss smth here.

I look at it and I see that this line does exactly the same as in `docs-development.js`

It iterates over loaders, not the files:
```js
config.module.loaders
           .map(value // value is loader
```

Isn't it ?